### ISSUE TITLE
Manual Proxy not working with proxy on same host.

### ISSUE THREAD

**Issue by dacoursey** [rNeomy/proxy-switcher#38]
Firefox 57.0
MacOS 10.12.6
PSM 0.1.8

Everything else seems to be working as expected. When I try to configure a Manual proxy to hit 127.0.0.1:8080 I never see any traffic hit that proxy.

I know the internal proxy is working because I use it with other apps every day. I tried setting the FF preferences to Use System Proxy as stated in other issues but it does not help.

<img width="503" alt="screen shot 2017-11-14 at 3 03 36 pm" src="https://user-images.githubusercontent.com/5431221/32802118-18779300-c94d-11e7-89b5-03edf2ed2907.png">

**Comment by ealdraed** [comment]
Hello @dacoursey !

I guess you should remove the `127.0.0.1` under `Direct:`. With your current config it will not use the manual proxy for `localhost, 127.0.0.1`, or in other words will access them directly.

**Comment by rNeomy** [comment]
@dacoursey are you still experiencing this problem?

**Comment by dacoursey** [comment]
Yes, removing the info from `Direct` didn't change the behavior.

Firefox just says `Connecting...` at the bottom and my local proxy never sees a request.

**Comment by rNeomy** [comment]
> Yes, removing the info from Direct didn't change the behavior

You are not supposed to remove. What is the IP address of the server? If for instance it is 192.168.1.1, then add the exact IP address in the "Direct" list or add something like "192.168.1.*" for instance.

**Comment by rNeomy** [comment]
> 127.0.0.1:8080 I never see any traffic hit that proxy.

Are you sure this is a SOCKS server? Usually 8080 is used for HTTP or HTTPS servers.

**Comment by appa1** [comment]
i have also this issue everything is perfect but does not receive any request in my burp
